django/core/cache/backends/redis.py

```diff
@@ -28,6 +30,8 @@ def loads(self, data):
 
 
 class RedisCacheClient:
+    _pools = {}
```

Comment:
This makes the value a _mutable class attribute_ shared across _all_ `RedisCacheClient` instances, where previously it was instance-specific, I think?
That might be fine/correct (I genuinely don't know this code _at all_, or how it's then used everywhere), but I'm just flagging it for scrutiny.

Reply:
I am also deeply unfamiliar with this section. I found the PR that introduced it: https://github.com/django/django/pull/14437
Perhaps @ngnpope or @carltongibson has an opinion :pray:

Reply:
@sarahboyce How can we ensure a progress in this PR? // @ngnpope @carltongibson

Reply:
[On our wiki](https://code.djangoproject.com/wiki/Reports#Reviewingpatches), this is a link to the [PR review queue](https://code.djangoproject.com/query?has_patch=1&needs_better_patch=0&needs_docs=0&needs_tests=0&stage=Accepted&status=!closed&order=changetime&desc=1). We also have some [FAQ docs](https://docs.djangoproject.com/en/5.1/faq/contributing/) about code contributions which include requesting for/waiting for a review.

As this is in the queue, the community can see it needs a review. All that's left is mostly being patient.

Most community members (including those tagged) are unpaid volunteers who have work and life commitments beyond this project. The deadline to get this bug fix merged in time to be included in the next release (Django 5.2) is February 19th 2025. There's no rush :heart: